plutus-contract/src/Wallet/Types.hs

```diff
@@ -117,6 +119,15 @@ contractInstanceIDs = ContractInstanceId <$> UUID.mockUUIDs
 randomID :: IO ContractInstanceId
 randomID = ContractInstanceId <$> UUID.nextRandom
 
+data ContractActivityStatus = Active | Stopped | Done deriving (Eq, Show, Generic, ToJSON, FromJSON, OpenApi.ToSchema)
```

Comment:
Could we just re-use the `Activity` type from `ContractInstance.STM` ?

Reply:
I believe it's better to separate API and domain types as they tend to change over time and it breaks the API without notices.